mkdocs.yml

```diff
@@ -26,6 +36,9 @@ plugins:
             docstring_style: numpy         
             show_source: true
             members_order: source
+            inherited_members: true
+          filters:
+          - "!neurostatslib.glm.GLMBase.residual_deviance"
```

Comment:
why do this? isn't `residual_deviance` under `NoiseModel`, not `GLMBase`?

Reply:
good catch, it is a leftover from the previous implementation